src/components/Notification/NotificationTray.react.js

```diff
@@ -0,0 +1,67 @@
+// @flow
+import * as React from "react";
+import { Notification, Dropdown } from "../";
+
+type Props = {|
+  /**
+   * Notification components
+   */
+  +children?: React.ChildrenArray<React.Element<typeof Notification>>,
+  /**
+   * An array containing objects of notification data
+   */
+  +notificationsObjects?: Array<{|
```

Comment:
Let's export this object type from `Notification` that way if the type changes we won't have to update both files